README.md

```diff
@@ -187,7 +187,26 @@ cmake .. -G Ninja -DCMAKE_INSTALL_PREFIX=$PWD/../bin -DCMAKE_BUILD_TYPE=Release
 cmake --build . --target install
 ```
 
-### 4) Create desktop shortcut or start AstroMenace
+#### Desktop Entry and Icon Installation
+
+On Linux, the build system can optionally install desktop entry and icon files following the Freedesktop standard. This is disabled by default.
+
+The installation includes:
+- Desktop entry file: `${CMAKE_INSTALL_PREFIX}/share/applications/astromenace.desktop`
+- Icon (64x64): `${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/64x64/apps/astromenace.png`
+- Icon (128x128): `${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/128x128/apps/astromenace.png`
+
+**To enable desktop file installation:**
+```
+cmake .. -DINSTALL_DESKTOP_FILES=ON
+```
+
+**For non-FHS systems (e.g., NixOS), customize installation paths:**
+```
+cmake .. -DINSTALL_DESKTOP_FILES=ON -DDESKTOP_FILE_DIR=/custom/path/applications -DICON_DIR=/custom/path/icons/hicolor

```

Comment:
@fgaz @trofi Does that make sense? It might be sufficient to simply have a single location, the `bin/share` folder, customizable. :thinking: